### PR TITLE
i18n improvement

### PR DESCRIPTION
**Pull request by benoitbar**
I worked on i18n improvement in models and template with ugettext and trans and I improve french transaltion in .po files.

### CODE REVIEW

File: wagtail/wagtailusers/models.py [hunk 1]
```diff
@@ -11,16 +11,19 @@ class UserProfile(models.Model):
     user = models.OneToOneField(settings.AUTH_USER_MODEL)
 
     submitted_notifications = models.BooleanField(
+        verbose_name=_('submitted notifications'),
```
Comment (kaedroho) [inline wagtail/wagtailusers/models.py]: These would probably require a migration

**Comment by kaedroho** [comment]
Merged i18n improvements in: d397dd1ef49e63b5d0d3a279ad03393c06f057a6

**Comment by kaedroho** [comment]
Merged french translations in: 41478e1faff047afdba13c446c65684e46f46ebc

Thanks!

**Comment by SalahAdDin** [comment]
regenerate all files for Transifex, i can help you with Spanish translation.

**Pull request closed without merging**